File: trunk/src/dotnet/yaxgl/yaxgl/EventHandler.cs
using System;
using System.Collections.Generic;
using System.Text;


namespace de.yaxgl
{
    [System.AttributeUsage(System.AttributeTargets.Method)]
    public class EventHandler : System.Attribute
    {
        public string eventID;
        public EventType eventType;

        public EventHandler(EventType eventType, string eventID)
        {
            this.eventID = eventID;
            this.eventType = eventType;
        }

    
    }
         
}

File: src/java/YaxglLibrary/de/yaxgl/EventDispatcher/Attribute/EventHandler.java
package de.yaxgl.EventDispatcher.Attribute;

import static java.lang.annotation.ElementType.*;

import java.lang.annotation.Retention;
import java.lang.annotation.*;

@Target(METHOD)
@Retention(RetentionPolicy.RUNTIME)
public @interface EventHandler {
	
	public EventType eventType();
	public String eventID();
	public boolean regexOn();
	
	//todo: wie ehier schon regexOn=false setzten standart
	//und gehen auch so konstruktoren wi in c#
}
